### ISSUE TITLE
Couldn't find any specified files, exiting without doing anything.

### ISSUE THREAD

**Issue by playground** [marcules/karma-remap-istanbul#2]
Report is not generated as if it can't see any files even though all tests executed.  Am I missing some settings?

Chrome 49.0.2623 (Mac OS X 10.11.2): Executed 62 of 63 (skipped 1) SUCCESS (3.278 secs / 3.271 secs)
Couldn't find any specified files, exiting without doing anything.

Here is my karma.config

```
files: [
  './src/testing/settings.js',
  'gulp-bootstrap.js',
  './src/**/*.js'
],
exclude: [
  './src/components/**/*.min.js'
],
jspm: {
  //packages: 'jspm_packages/',
  loadFiles: ['./src/**/*.spec.js'],
  serveFiles: ['./src/**/*.js', './src/**/*.map', './src/**/*.ts']
},
reporters: ['progress', 'karam-coverage', 'karma-remap-istanbul'],
remapIstanbulReporter: {
  src: 'coverage/report.json',
  reports: {
    lcovonly: 'coverage/lcov.info',
    html: 'coverage/html/report'
  },
  timeoutNotCreated: 1000, // default value
  timeoutNoMoreFiles: 1000 // default value
},
preprocessors: {
  './**/*.js': ['karma-coverage', 'coverage']
},
```

**Comment by marcules** [comment]
``` js
reporters: ['progress', 'karam-coverage', 'karma-remap-istanbul'],
```

Is `karam-coverage` actually from your karma.config or just a copy & paste error? Does the file you specify at `coverage/report.json` exist?

Just a guess, but maybe you just forgot to configure karma itself, here's one of my configs:

``` js
    coverageReporter: {
      type: 'json',
      dir: 'reports',
      subdir: 'coverage',
      file: 'coverage-final.json'
    },
```

Maybe that's already the solution :)

**Comment by playground** [comment]
Will you share with me your working karma.config?

I'm sure I'm doing something wrong here.

**Comment by marcules** [comment]
Sure, this is one that's working for me

``` js
'use strict';

var path = require('path');
var conf = require('./gulp/conf');

module.exports = function (config) {

  var configuration = {
    autoWatch: false,

    frameworks: ['jasmine'],

    ngHtml2JsPreprocessor: {
      stripPrefix: 'src/',
      moduleName: 'gulpAngular'
    },

    browsers: ['PhantomJS'],

    plugins: [
      'karma-phantomjs-launcher',
      'karma-jasmine',
      'karma-ng-html2js-preprocessor',
      'karma-coverage',
      'karma-sonarqube-unit-reporter',
      'karma-remap-istanbul'
    ],

    preprocessors: {
      'src/**/*.html': ['ng-html2js']
    },

    reporters: [
      'progress',
      'sonarqubeUnit',
      'coverage',
      'karma-remap-istanbul'
    ],

    sonarQubeUnitReporter: {
        outputFile: 'reports/ut_report.xml',
        useBrowserName: false
    },

    coverageReporter: {
      type: 'json',
      dir: 'reports',
      subdir: 'coverage',
      file: 'coverage-final.json'
    },

    remapIstanbulReporter: {
      src: 'reports/coverage/coverage-final.json',
      reports: {
        lcovonly: 'reports/coverage/lcov.info',
        html: 'reports/coverage/html'
      }
    }
  };
  configuration.preprocessors[path.join(conf.paths.tmp, 'serve/**/*.js')] = ['coverage'];

  // This block is needed to execute Chrome on Travis
  // If you ever plan to use Chrome and Travis, you can keep it
  // If not, you can safely remove it
  // https://github.com/karma-runner/karma/issues/1144#issuecomment-53633076
  if (configuration.browsers[0] === 'Chrome' && process.env.TRAVIS) {
    configuration.customLaunchers = {
      'chrome-travis-ci': {
        base: 'Chrome',
        flags: ['--no-sandbox']
      }
    };
    configuration.browsers = ['chrome-travis-ci'];
  }

  config.set(configuration);
};

```

But my setup is a bit different than your's I'm guessing, so you won't be able to just copy-paste it.

**Comment by playground** [comment]
@marcules I finally got it working.  Thanks.

**Comment by Kavorka79** [comment]
I have a karma.conf.js that look similar to yours but I still get:  **Couldn't find any specified files, exiting without doing anything.**

coverage-final.json is created but remap fail.
I have tried the timeouts but no success...

```
        // test results reporter to use
        // possible values: 'dots', 'progress'
        // available reporters: https://npmjs.org/browse/keyword/karma-reporter
        reporters: ["progress", "coverage", "karma-remap-istanbul"],

        // default value
        coverageReporter:
        {
            includeAllSources: true,
            type: "json",
            dir: "coverage",
            subdir: "json",
            file: "coverage-final.json"
        },
        remapIstanbulReporter: {
            src: "coverage/json/coverage-final.json",
            reports: {
                html: "coverage/html"
            }
        },
```

**Comment by delasteve** [comment]
Could one of you having this issue create a simple repository that shows this behavior?

**Comment by MuhammedHasan** [comment]
I have this issue in [my project](https://github.com/MuhammedHasan/metabol/tree/develop).

When I run my code first time, it generates coverage-final.json. Then it generates remap report file in the second run with previous coverage-final.json.

Thus, it gives this error in the first run because it can not find coverage-final.json.

**Comment by delasteve** [comment]
Should be fixed in `0.2.1`.
